server/routes/serviceProviderReferrals/editSessionForm.ts

```diff
@@ -24,8 +24,8 @@ export default class EditSessionForm {
     return {
       error: null,
       paramsForUpdate: {
-        appointmentTime: dateResult.value.toISOString() ?? null,
-        durationInMinutes: durationResult.value.minutes ?? null,
+        appointmentTime: dateResult.value.toISOString(),
+        durationInMinutes: durationResult.value.minutes!,
```

Comment:
is this ok to do? does the validation step check that this will not be null?

Reply:
The UI prevents form submission if any of those fields are empty, so it _should_ be safe

Reply:
there's a test with `'duration-minutes': '',` input and it seems to pass 🤷‍♂️